### PR TITLE
Advantage Air Documentation

### PR DESCRIPTION
**Pull request by Bre77**
## Proposed change

This is an integration for the Advantage Air range of ducted air-conditioning systems controlled by a wall mounted Android tablet. The products have various names such as MyAir, e-zone, and zone10e. The integration creates Climate entities, Cover entities, Sensors, and Binary Sensors dynamically based on what hardware is exposed by the product.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/40159
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1864
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Bre77** [comment]
This can be closed if https://github.com/home-assistant/home-assistant.io/pull/15049 is accepted instead.

**Review by frenck — APPROVED** [review]
Thanks, @Bre77 👍

File: source/_integrations/advantage_air.markdown [hunk 1]
```diff
@@ -49,13 +49,13 @@ The `advantage_air` binary sensor platform will create a binary sensor for each
...
-### set_time_to
+### Service `advantage_air.set_time_to`
 
 Set the On/Off Timer using the relevant sensor entity.
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | no | `sensor.[name]_time_to_on` or `sensor.[name]_time_to_off`
-| `entity_id` | no | Number of minutes between `0` and `720`.
+| `entity_id` | yes | `sensor.[name]_time_to_on` or `sensor.[name]_time_to_off`
```
Comment (Bre77) [inline source/_integrations/advantage_air.markdown:60]: Hey @frenck, entity_id is not optional

> Failed to call service advantage_air/set_time_to. must contain at least one of entity_id, area_id.

Comment (frenck) [reply source/_integrations/advantage_air.markdown:60]: It is optional. It is registered using: `async_register_entity_service`, defined here:

<https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_platform.py#L543>

Which uses:

https://github.com/home-assistant/core/blob/00847db54f00a821f044243696a09638a1847aa2/homeassistant/helpers/config_validation.py#L858-L874

Which has: `vol.Optional(ATTR_ENTITY_ID)`.

Alternatively, these services can be targeting an area of entities.

Comment (Bre77) [reply source/_integrations/advantage_air.markdown:60]: Ahh yes, I understand now. Thank you.

**Pull request merged**